.env.kesaseteli.example

```diff
@@ -24,5 +24,10 @@ ADFS_TENANT_ID=
 ADFS_LOGIN_REDIRECT_URL=
 ADFS_LOGIN_REDIRECT_URL_FAILURE=
 
+# Django-storages
+AZURE_ACCOUNT_NAME=
+AZURE_ACCOUNT_KEY=
+AZURE_CONTAINER=
```

Comment:
Add something to e.g. README, how to determine these values, and how to run locally with file system storage?

Reply:
I put this in the settings so locally this should not require any changes:
```
DEFAULT_FILE_STORAGE=(str, "django.core.files.storage.FileSystemStorage"),
```

I'll add a mention to the README about the AzureStorage.